src/script/lua_api/l_metadata.cpp

```diff
@@ -229,6 +229,56 @@ int MetaDataRef::l_get_keys(lua_State *L)
 	return 1;
 }
 
+// get_bool(self, name)
+int MetaDataRef::l_get_bool(lua_State *L)
+{
+	MAP_LOCK_REQUIRED;
+
+	MetaDataRef *ref = checkAnyMetadata(L, 1);
+	std::string name = luaL_checkstring(L, 2);
+
+	IMetadata *meta = ref->getmeta(false);
+	if (meta == NULL) {
+    	return luaL_error(L, "The boolean meta value can't be nil");
+	}
```

Comment:
this error message feels confusing. When would this error be raised?

Reply:
It is raised whenever it accesses a nil value, exactly like the docs say